Tools/AP_Bootloader/board_types.txt

```diff
@@ -225,6 +225,8 @@ AP_HW_SDMODELH7V1                    1111
 AP_HW_FlyingMoonH743                 1112
 AP_HW_YJUAV_A6                       1113
 AP_HW_YJUAV_A6Nano                   1114
+AP_HW_ACNS_CM4PILOT                  1115
```

Comment:
If you want to reserve these IDs, make a separate PR.  Otherwise someone might get in ahead of you.